.azure-pipelines/azure-pipelines-linux.yml

```diff
@@ -5,7 +5,7 @@
 jobs:
 - job: linux
   pool:
-    vmImage: ubuntu-16.04
+    vmImage: ubuntu-20.04
```

Comment:
https://docs.microsoft.com/en-us/azure/devops/release-notes/2021/pipelines/sprint-193-update#updated-schedule-for-removal-of-ubuntu-1604-image-on-microsoft-hosted-agents